arrays/arrays.c

```diff
@@ -59,7 +59,7 @@ void resize_array(Array *arr) {
 
   // Create a new element storage with double capacity
   int doubleCapacity = arr->capacity * 2;
-  char **newElements = realloc(arr->elements, doubleCapacity * sizeof(char *));
+  char **newElements = calloc(doubleCapacity, sizeof(char *));
```

Comment:
Good, this is how I'd recommend doing this if you had to choose between calloc() and realloc() again, I don't think there is anything strictly wrong with using realloc().